packages/model-instance-client/src/index.ts

```diff
@@ -1,8 +1,8 @@
 export {
   ModelInstanceClient,
   type PostDataParams,
-  type PostDeterministicInitParams,
-  type PostSignedInitParams,
+  type CreateSingletonParams as PostDeterministicInitParams,
```

Comment:
Do we want to export them under the new name as well? Or make the breaking change now use the new names publicly?

Reply:
Yes we want to change them everywhere, I missed this one. The LSP rename feature does this which is a bit annoying. I'll fix it .